### PR TITLE
Feature/113 adjust fees

### PR DESCRIPTION
**Pull request by kevenboo**
related ticket ->https://github.com/lncapital/torq/issues/113

- Updated channel list backend end front end to fetch and display the fees
- Updated the updateChannel endpoint
- Added the Update fee flow in the front end
Screen Shot 2022-10-19 at 3 45 52 PM
Screen Shot 2022-10-19 at 3 46 40 PM
Screen Shot 2022-10-19 at 3 46 50 PM

Errors
Screen Shot 2022-10-19 at 3 47 13 PM

### CODE REVIEW

File: web/src/features/channels/updateChannel/UpdateChannelModal.tsx [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+  const { data: localNodes } = useGetLocalNodesQuery();
+  let { data: channels } = useGetChannelsQuery();
+
+  let localNodeOptions: SelectOptions[] = [{ value: 0, label: "Select a local node" }];
+  if (localNodes !== undefined) {
+    localNodeOptions = localNodes.map((localNode: localNode) => {
+      return { value: localNode.localNodeId, label: localNode.grpcAddress };
+    });
+  }
```
Comment (maxwedwards) [inline web/src/features/channels/updateChannel/UpdateChannelModal.tsx:51]: I think this should go inside a `useEffect()`

File: web/src/features/channels/updateChannel/UpdateChannelModal.tsx [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+  let channelOptions: SelectOptions[] = [{ value: 0, label: "Select your channel" }];
+  if (channels !== undefined) {
+    channelOptions = channels.map((channel: channel) => {
+      return { value: channel.lndShortChannelId, label: channel.lndShortChannelId.toString() };
+    });
+  }
```
Comment (maxwedwards) [inline web/src/features/channels/updateChannel/UpdateChannelModal.tsx:57]: I think this should go inside a `useEffect()`

**Review by maxwedwards — COMMENTED** [review]
Looks great, just a couple of places where I think the code should be inside a `useEffect()`

**Comment by ohenrik** [comment]
This looks good, but I have a question about the UX. Can a user click on a "update fees" button on each channel on the channels page so that the channel selector is already filled in? If not that should probably be added. Another thing is that we should have a list of peer aliases that we can use in the channel selector (with the channel ID after it) That would make it much easier to select the correct channel.

**Comment by kevenboo** [comment]
> This looks good, but I have a question about the UX. Can a user click on a "update fees" button on each channel on the channels page so that the channel selector is already filled in? If not that should probably be added. Another thing is that we should have a list of peer aliases that we can use in the channel selector (with the channel ID after it) That would make it much easier to select the correct channel.

- We don't have a button next to each channels. i can look into it. You want it on this PR or you want in another one ?
- Ok I will check as well for the peerAlias

File: web/src/features/channels/updateChannel/UpdateChannelModal.tsx (hunk 2 above)
Comment (ohenrik) [reply web/src/features/channels/updateChannel/UpdateChannelModal.tsx:57]: @maxwedwards I quickly tested this out and it did not seem to work. I can look into refactoring this as part of the new invoice modal.

Comment (ohenrik) [reply web/src/features/channels/updateChannel/UpdateChannelModal.tsx:57]: I'll merge this for now

**Comment by ohenrik** [comment]
> * We don't have a button next to each channels. i can look into it. You want it on this PR or you want in another one ?
> * Ok I will check as well for the peerAlias

We can merge this in first and I'll design and add a button in a new PR.

**Comment by ohenrik** [comment]
Closes #113

**Pull request merged**